src/platform/plugins/shared/dashboard/server/api/transforms/out/transform_options_out.test.ts

```diff
@@ -23,11 +23,29 @@ describe('transformOptionsOut', () => {
     });
     const result = transformOptionsOut(optionsJSON);
     expect(result).toEqual({
-      hidePanelTitles: true,
+      hide_panel_titles: true,
     });
     expect(result).not.toHaveProperty('unsupportedOption');
   });
 
+  it('should translate all options', () => {
+    const optionsJSON = JSON.stringify({
+      hidePanelTitles: true,
```

Comment:
Same here, lots have one value be `false` to prevent any truthy logic errors in future.